### ISSUE TITLE
pixiv | pixiv.net

### ISSUE THREAD

**Issue by claire-rouge** [PreMiD/Presences#302]
**What's the name of the service?**
https://www.pixiv.net/en/ | pixiv

**What should the Presence display?**
Informations about the current page:

1. > Browsing Artwork / Novel / Group, 
2. > Profile name / Group name.

and (when you are not on a profile/group but is on site in general.)

easy route: 
1. > Browsing site
2. > on pixiv.net

or more depth: 
1. > Browsing feed / Browsing New Work: Following / Browsing New Work: Everyone / Browsing bookmarks / Browsing Top Rankings.
2. > on pixiv.net

Top Rankings has; Daily, Weekly, Monthly, Rookie, Original, Popular ranking (Male members), Popular ranking (Female members), General, R-18
So it would best to just slot all these under one like: Browsing Top Rankings.

based how the information would show like the twitter one.
![](https://files.catbox.moe/yr6oj5.png)
Illustrations, Ugoira(Animations), Manga goes under: Browsing Artwork.
Novel goes under: Browsing Novel
(Novel is the only one that has link address have /novel/ in it, the rest use /artworks/, could use this to group stuff together)

link to show what I mean
Illustration https://www.pixiv.net/en/artworks/77066151
Ugoira(Animations) https://www.pixiv.net/en/artworks/77074846
Manga https://www.pixiv.net/en/artworks/67078695
Novel https://www.pixiv.net/novel/show.php?id=11750989
![](https://files.catbox.moe/gcqguc.png)

### TLDR: do whats easy and dont listen to most of what I have listed

**Why do you want support for this service?**
pixiv is a big social media service where users can post their works (e.g. illustrations, manga, novels) and receive much love and support! from fans, you could say its the japanese version of deviant art but there isnt only japanese artist on there, western users are there also.

It would be nice to share this passion with others on Discord!

**Are you able to provide an image for the service? (512x512 minimum)**
Download link to logo and icons https://s.pximg.net/www/images/terms/logo_pixiv_20181115.zip
Here is how all the icon looks (1000x1000)
![](https://files.catbox.moe/fjcwj0.png)

**Info about the page**
Pixiv (Japanese: ピクシブ Hepburn: Pikushibu) is a Japanese online community for artists. It was first launched as a beta test on September 10, 2007 by Takahiro Kamitani and Takanori Katagiri. ... Pixiv aims to provide a place for artists to exhibit their work(e.g. illustrations, manga, novels) and get feedback via a rating system and user comments.

**Comment by OfficialCRUGG** [comment]
Will do later today or tomorrow!
